no-envelopes-processed-alert.tf

```diff
@@ -20,7 +20,7 @@ traces
 | project ["# files"],
     interval_start = last_event_time > datetime("2018-02-02 09:59:00"),
     interval_end = last_event_time < datetime("2018-02-02 17:01:00"),
```

Comment:
unrelated to the change in this PR, but what are those hardcoded dates?

Reply:
reference #76 ;)